db/db.go

```diff
@@ -127,7 +127,7 @@ func GetEth1Deposits(address string, length, start uint64) ([]*types.EthOneDepos
 
 var searchLikeHash = regexp.MustCompile(`^0?x?[0-9a-fA-F]{2,96}`) // only search for pubkeys if string consists of 96 hex-chars
 
-func GetEth1DepositsJoinEth2Deposits(query string, length, start uint64, orderBy, orderDir string, latestEpoch, validatorOnlineThresholdSlot uint64) ([]*types.EthOneDepositsData, uint64, error) {
+func GetEth1DepositsJoinEth2Deposits(query string, length, start uint64, orderBy, orderDir string, latestEpoch, validatorOnlineThresholdEpoch uint64) ([]*types.EthOneDepositsData, uint64, error) {
```

Comment:
Isn't it enough to show a validator as offline after more than one finalized epoch instead of more than two?

Reply:
Quoting the BIDS' description: `The "Active" status on the validator page and dashboard, should only turn red if the validator was not attesting for at least 2 finalized epochs.`

If the last finalized epoch is 1000, the active status should turn red if the latest attestation happened in or before epoch 998. The **validatorOnlineThesholdEpoch** in that case is set to 999 any by comparing it with '<', this should be correct, should it not?

Reply:
Yes in the bids it says at least 2 finalized epochs. Although in my opinion that is an overkill. If you pass the latestEpoch and the latestFinalizedEpoch to the GetEth1DepositsJoinEth2Deposits function you don't need the validatorOnlineThresholdEpoch. In my opinion it would be cleaner to have an inequality:
`lastattestationslot / $5 < $4` where $5 is the slots per epoch and $4 is the lastFinalizedEpoch.

There are always at least 1 - 2 epochs above the latest finalized epoch and using a value of at least 1 finalzed epoch would mean that the validator has failed to attest in at least 2(-3) epochs.

Reply:
For example, if the head epoch is 1000 and the latestFinalized epoch is 998 the validator has a chance to attest in epoch 1000, 999, 998 before being considered offline.